pkgs/development/tools/convox/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, fetchurl, buildGoPackage }:
+
+buildGoPackage rec {
+  name = "convox-${version}";
+  version = "20171002150509";
+
+  meta = {
+    homepage = https://convox.com;
+    description = "Open-source PaaS";
+    license = lib.licenses.asl20;
+    platforms = lib.platforms.linux ++ lib.platforms.darwin;
+  };
+
+  goPackagePath = "github.com/convox/rack";
+  subPackages = [ "cmd/convox" ];
+
+  src = fetchurl {
```

Comment:
You can use `fetchFromGitHub` instead, so import `fetchFromGitHub` instead of `fetchurl` at the top, and:

``` nix
{
  src = fetchFromGitHub {
    owner = "convox";
    repo = "rack";
    rev = version;
    sha256 = "05hxrb36b9fxgid3n43z43k4zw11akn80kggcxkxx4ba3z6pff8q";
  };
}
```